### PR TITLE
using Blah -> import Blah as BL

### PR DESCRIPTION
**Pull request by trontrytel**
This solves #20

### CODE REVIEW

**Comment by trontrytel** [comment]
Looks like the nice syntax doesn't work in julia 1.5. Will fix that

File: src/Common.jl [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
-
-import CLIMAParameters as CP
-import CLIMAParameters.Planet as CP_planet
-import CLIMAParameters.Atmos.Microphysics as CP_micro
+import Thermodynamics
+import CLIMAParameters
+import CLIMAParameters.Planet
+import CLIMAParameters.Atmos.Microphysics
+
+TD = Thermodynamics
```
Comment (jakebolewski) [inline src/Common.jl:11]: you want to declare these as `const`

Comment (trontrytel) [reply src/Common.jl]: Makes sense! done

**Review by charleskawczynski — APPROVED** [review]
LGTM

**Pull request merged**